src/items.rs

```diff
@@ -1240,11 +1240,55 @@ pub fn rewrite_associated_type(ident: ast::Ident,
     let type_bounds_str = if let Some(ty_param_bounds) = ty_param_bounds_opt {
         let bounds: &[_] = ty_param_bounds;
         let bound_str = try_opt!(bounds.iter()
-                                     .map(|ty_bound| {
-            ty_bound.rewrite(context, Shape::legacy(context.config.max_width, indent))
-        })
-                                     .intersperse(Some(" + ".to_string()))
-                                     .collect::<Option<String>>());
+            .map(|ty_bound| {
+                ty_bound.rewrite(context, Shape::legacy(context.config.max_width, indent))
+            })
+            .intersperse(Some(" + ".to_string()))
+            .collect::<Option<String>>());
```

Comment:
It looks like this is just a reformatting change. It is nice (though not 100% necessary) to keep reformatting changes in a separate commit because when changing rustfmt there can be lots of them.